lib/foreman/renderer.rb

```diff
@@ -274,7 +274,7 @@ def jumpstart_attributes
     end
 
     def kickstart_attributes
-      @dynamic   = @host.diskLayout =~ /^#Dynamic/ if (@host.respond_to?(:disk) && @host.disk.present?) || @host.ptable.present?
+      @dynamic   = @host.disk_layout_template[:template] =~ /^#Dynamic/ if @host.disk_layout_template
```

Comment:
`.start_with('#Dynamic')` is faster and more readable then regex matching.

Reply:
updated